07-http_gordle/2_2_endpoints_chi/internal/handlers/newgame/handler.go

```diff
@@ -4,15 +4,15 @@ import (
 	"encoding/json"
 	"net/http"
 
-	"learngo-pockets/httpgordle/api"
+	"learngo-pockets/httpgordle/internal/api"
 )
 
 // Handle is the handler for the game creation endpoint.
 func Handle(w http.ResponseWriter, req *http.Request) {
 	apiGame := api.GameResponse{}
 
-	w.WriteHeader(http.StatusCreated)
 	w.Header().Set("Content-Type", "application/json")
```

Comment:
Fun-fact: This line is not necessary:
```
	// If the Header
	// does not contain a Content-Type line, Write adds a Content-Type set
	// to the result of passing the initial 512 bytes of written data to
	// DetectContentType. 
```
I would keep the call to Set("Content-type") though, as we are not explicitly calling `Wrtite()`